lightning/src/ln/channelmanager.rs

```diff
@@ -754,22 +754,43 @@ macro_rules! handle_error {
 	}
 }
 
+macro_rules! convert_chan_err {
+	($self: ident, $err: expr, $short_to_id: expr, $channel: expr, $channel_id: expr) => {
+		match $err {
```

Comment:
For readability, it would be helpful to use a variable here to know what the bool is meant to represent. Or add some documentation on the what the macro evaluates to.